src/lib/metrics.ts

```diff
@@ -92,16 +92,15 @@ class MetricsMonitor {
         });
 
         clientMetricsStore.on('metrics', m => {
-            // eslint-disable-next-line no-restricted-syntax
-            for (const [feature, { yes, no }] of Object.entries(
+            for (const entry of Object.entries(
```

Comment:
I couldn't get the expansion working in TS so I have to manually pull out the values. I suspect it will behave better when we add the upstream types